### PR TITLE
booting.md: Fix x86_64 boot explanation

### PR DESCRIPTION
**Pull request by gettsu**
adapted the file and function names to the current repository

### CODE REVIEW

File: content/en/docs/develop/booting.md [hunk 1]
```diff
@@ -8,24 +8,26 @@ description: |
...
 
-### x86_64 
+### x86_64
 
-The next step is done by the bootloader, `Multiboot` in our case, which sets up some of the sections (zero-ing the `.bss`, writing the `.text` with the effective code) and passes information about the memory layout and available devices to the entry function, `_libkvmplat_start32`.
+The next step is done by the bootloader, `Multiboot` in our case, which sets up some of the sections (zero-ing the `.bss`, writing the `.text` with the effective code) and passes information about the memory layout and available devices to the entry function, `lcpu_start32`.
 The CPU is left in the `Protected Mode`, and a small piece of code is needed to enable the `Long Mode`, paging and extended instruction sets, like `AVX`.
-All those are done in `plat/kvm/x86/entry64.S`.
-After that, interrupts, traps, the heap, the ACPI tables, and others, are initialized, in the `_libkvmplat_entry` function, in `plat/kvm/x86/setup.c`.
+All those are done in `plat/kvm/x86/lcp_start.S`.
```
Comment (StefanJum) [inline content/en/docs/develop/booting.md:22]:
```suggestion
All those are done in `plat/kvm/x86/lcpu_start.S`.
```

File: content/en/docs/develop/booting.md [hunk 2]
```diff
@@ -8,24 +8,26 @@ description: |
...
+After that, interrupts, traps, the heap, the ACPI tables, and others, are initialized, in the `ukplat_entry` function, in `plat/kvm/x86/setup.c`.
```
Comment (StefanJum) [inline content/en/docs/develop/booting.md:23]:
```suggestion
After that, interrupts, traps, the heap, the ACPI tables, and others, are initialized, in the `_ukplat_entry` function, in `plat/kvm/x86/setup.c`.
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
Looks good @gettsu, besides the few comments, please also squash your commits.

**Comment by gettsu** [comment]
Thanks for your comments !

**Review by StefanJum — CHANGES_REQUESTED** [review]
Looks better and better, thanks @gettsu.
Please sign-off your commit and add a short description on why the informations were outdated.

**Comment by StefanJum** [comment]
@gettsu all looks good now, thanks a lot.
Please modify your commit message to fit to ~72 characters per line then we can merge this.

**Comment by gettsu** [comment]
@StefanJum Thanks a lot! I just fixed it!

**Review by StefanJum — APPROVED** [review]
All good now, thanks @gettsu.

Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**